plugins/br.com.embraer.massif.commandevaluation.client/META-INF/MANIFEST.MF

```diff
@@ -5,8 +5,8 @@ Bundle-SymbolicName: br.com.embraer.massif.commandevaluation.client
 Bundle-Version: 0.7.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Require-Bundle: br.com.embraer.massif.commandevaluation.base;bundle-version="0.4.0";visibility:=reexport,
- com.sun.jna;bundle-version="4.5.1",
- com.sun.jna.platform;bundle-version="4.5.1"
+ com.sun.jna;bundle-version="4.1.0",
```

Comment:
The target used by the build should also be downgraded if an earlier version is also acceptable:
https://github.com/viatra/massif/blob/master/releng/hu.bme.mit.massif.releng/hu.bme.mit.massif.releng.target#L23